hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -601,6 +605,16 @@ public OzoneOutputStream createKey(
     HddsClientUtils.verifyResourceName(volumeName, bucketName);
     HddsClientUtils.checkNotNull(keyName, type, factor);
     String requestId = UUID.randomUUID().toString();
+
+    if(Boolean.valueOf(metadata.get(OzoneConsts.GDPR_FLAG))){
+      try{
+        GDPRSymmetricKey gKey = new GDPRSymmetricKey();
+        metadata.putAll(gKey.getKeyDetails());
+      }catch (Exception e) {
+        throw new IOException(e);
```

Comment:
@ajayydv The only time this line would create an exception is when the host does not have JCE policy jars installed as the default secret is 32 chars and without the JCE policy jars it would throw "java.security.InvalidKeyException: Illegal key size or default parameters". Since we are throwing the exception and not bypassing it without successful key generation, a debug statement won't add much value.

Reply:
Yes, And see my comment above. The fact that Illegal key size is the result of Java Security policy is not very obvious. If you can add that to your throw or log statement, it would be really good. Then the user realizes why it is failing. The issue more precisely is that this is client code; we have no control over where or on what machines this code is run. It is quite possible that when the Ozone cluster is being setup the Admins installed the right policy; but some client machine may not have that. If the key has been written by a client with 256 bits length, then the new client has no choice but the use the same algorithm to decode it. Communicating that issue to the user might save some pain for them.

Reply:
@anuengineer thanks for elaborating, @ajayydv thanks for original suggestion.
I have added an ERROR message before throwing the exception.